.github/workflows/auto-badge.yaml

```diff
@@ -0,0 +1,55 @@
+name: auto-badge
+
+on:
+  workflow_call:
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+      SLACK_WEBHOOK:
+        required: true
+
+env:
+  REPO_NAME: ${{ github.event.repository.name }}
+  REPO_OWNER: ${{ github.repository_owner }}
+
+jobs:
+  auto-badge:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Git checkout
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Git checkout Labs Tools
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.AUTOMATION_TOKEN }}
+        repository: boldlink/labs-tools-scripts
+        ref: 'main'
+        path: .tmp
+
+    - name: Adding Workflow Badges
+      shell: bash
+      run: |
+        bash \
+        ${GITHUB_WORKSPACE}/.tmp/scripts/auto-badge/autobadge.sh \
+        && rm -rf ${GITHUB_WORKSPACE}/.tmp/
+
+    - name: Push changes automatically!
+      uses: EndBug/add-and-commit@v9
+      with:
+        message: '[Boldlinksig]: Workflow Badges Added/Updated to all README.md files.'
+        author_name: boldlinksig
+        author_email: boldlinksig@boldlink.io
+        committer_name: boldlinksig
+        committer_email: boldlinksig@boldlink.io
+    - name: Slack Notification failure
+      if: failure()
+      uses: rtCamp/action-slack-notify@v2
+      env:
+        SLACK_USERNAME: Boldlinksig
```

Comment:
Repo name is required, if this runs on many repos most likely we will endup with many alerts and without visually identify the failed workflows.